src/js/sonic-reveal.js

```diff
@@ -1,42 +1,42 @@
 
 let attr_texts = {
   danceability: [
-    `<a href="https://www.youtube.com/watch?v=n7RuIga3vdk">It's got a good beat, and you can dance to it!</a>`,
-    `The <b>danceability</b> score is a measure of how suitable a song is to dance to. “Believe” is a real banger, so it's a little surprising it wasn't rated 100% on this scale. However, it's important to note that these scores don't come from kids on American Bandstand - they come from a computer. Danceability is calculated algorithmically using criteria like the tempo and the strength and regularity of the beat.`,
-    `The latter might explain why this song didn't reach the top of the danceability scale. The song features two sections dominated by synth pads, with no percussion: the intro, and a breakdown following the bridge. Our algorithm is very likely to "lose the beat" during these sections, and therefore ding the danceability score.`
+    `It’s got a good beat, and you can dance to it!`,
```

Comment:
Maybe enclose in quotation marks? Or I guess it's not a huge deal if people don't get that it's a reference.

Reply:
Ha! Completely missed the reference myself. I can put the quotes back